### ISSUE TITLE
Guide Articles still marked as stub even when they have data

### ISSUE THREAD

**Issue by PragatiVerma18** [freeCodeCamp/freeCodeCamp#36086]
Many guide articles are still marked as stub even though they have been filled with appropriate data. See the photo below to consider a few for example:
![freecode](https://user-images.githubusercontent.com/42115530/57986345-5a93e800-7a91-11e9-8186-42c957ded33b.JPG)

**Comment by RandellDawson** [comment]
Yes, there are many challenges which do not have hints and solutions (hence the stub).  If you would like to help replace the stub with applicable hint(s) and solution(s), make sure to check this folder where all are kept.  It has been a while since the master branch has been deployed to production, so you may find files which actually have a solution but does not show on the production site.

**Mods:** This issue is related to issue #35890

**Comment by PragatiVerma18** [comment]
So, where exactly to look if someone wants to contribute by removing stubs and replacing them with some useful text?

Also, where to make changes so that that stub tag that appears in grey along the text can be removed, I have been looking although, couldn't find the appropriate folder.

Hope you can help, thanking you!!

**Comment by RandellDawson** [comment]
You can look in [this directory](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/guide/english/certifications) and find the applicable Guide article for the challenge.

Once the stub information is removed and replaced by your content, it would no longer be grayed out on the production site (once the PR is merged and the master branch is deployed to production).

Please read through [How to work on Guide articles](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/docs/how-to-work-on-guide-articles.md) and [Style guide for creating and editing Guide Articles](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/docs/style-guide-for-guide-articles.md).  If you have any questions you can ask in the [Contributors Gitter channel](https://gitter.im/FreeCodeCamp/Contributors).

**Comment by PragatiVerma18** [comment]
Thank you for the directory link. I found the applicable guide articles there. But I am still confused regarding the stub tag, I checked the [Linear Gradient](https://github.com/PragatiVerma18/freeCodeCamp/tree/master/guide/english/css/linear-gradient) guide article code, the stub text has been removed from there.

![2](https://user-images.githubusercontent.com/42115530/57996736-d10f0500-7ae6-11e9-8466-f33aac3e5692.JPG)

Also, I checked the last merge, which is as follows:
![freecode2](https://user-images.githubusercontent.com/42115530/57996575-e172b000-7ae5-11e9-98f1-de54b4085800.JPG)

It has been merged but maybe not deployed as it is still greyed out on the website:
![1](https://user-images.githubusercontent.com/42115530/57996724-c2c0e900-7ae6-11e9-890e-ebbe79aacb2b.JPG)
Hope you can clear my confusion. Sorry for the trouble.

**Comment by RandellDawson** [comment]
@PragatiVerma18 You are correct that it has been merged but has not been deployed.  The entire master branch has not been deployed to production since 2nd week of October.  There was a huge change the overall structure of the freeCodeCamp repo in October after the last deployment.  The core team is finalizing some things currently, so the master branch can be deployed to production, but there is no estimate of when that work will be completed.

**Comment by PragatiVerma18** [comment]
Thank you for the details. Hoping for the updates soon!!!

**Comment by moT01** [comment]
So do we want to have someone go through the guide and find all the articles that have content, but the stub text didn't get removed? That would probably be the solution to this - the article in the issue here has already been fixed, but there is probably others.

**Comment by RandellDawson** [comment]
@moT01 I can do that.  Should I create a new issue (referencing this one) and post a list of all the files meeting this criteria?

**Comment by moT01** [comment]
Whatever you want - this issue could just be transformed to deal with it. But either way works. I may have a script that can be slightly modified to achieve this.

**Comment by RandellDawson** [comment]
Let's update this issue then.  If you already have a script, then go ahead.  I also have a script which could be modified to accomplish it.  Just let me know.

**Comment by moT01** [comment]
The article mentioned in the issue here will be fixed on the next deployment as mentioned. Also, I went through and found no other articles on the master branch which have stubs and content. There might still be some on the current site, but should all be fixed when master is deployed. Thanks and happy coding everyone!
